src/app/ui/components/containers/headers/header.tsx

```diff
@@ -119,16 +113,7 @@ export function Header({
           <NetworkModeBadge isTestnetChain name="Testnet" onClick={() => null} />
           {variant !== 'onboarding' && (
             <>
-              {/* <Button
-                data-testid={SettingsSelectors.SettingsMenuBtn}
-                // FIXME 4370 task #6 - need to make this settings menu clickable here
-                // onClick={() => setIsShowingSettings(!isShowingSettings)}
-                onClick={() => null}
-                variant="ghost"
-              >
-                <HamburgerIcon />
-              </Button> */}
-              {/* <DropdownMenu.Root>
+              <DropdownMenu.Root>
```

Comment:
@fbwoolf  - in [this commit ](https://github.com/leather-wallet/extension/pull/4655/commits/3e1a9b48ee6dabb48d58a21dce1aafbc52d6d440) I have added a very basic prototype of the new settings menu embedded in the new `<Header`. 
![Screenshot 2024-01-22 at 11 12 04](https://github.com/leather-wallet/extension/assets/2938440/319e296c-a6be-493e-ade2-94301363e28b)

As I work, my plan will be to:
- deprecate `useHeader`
- move `<Header` logic to `app/features/container` 
- pass the Settings dropdown as a prop into `<Header` there

I don't think we need to use `routeHeader` anymore, or state to manage the header but I will see as I go. 

I deleted some things from the store as I think the new menu should be able to work without it. We may need to re-add that if I'm mistaken.

Reply:
Ok, so from this comment, I think you are tackling the settings menu for now and I should move onto the 'Items' issue I'm assigned to? Or, am I supposed to be working off this branch to help out in finishing the settings menu?

Reply:
I was just sharing how I intend to use it. If you have time to do it that would be great, if not I will get to it towards the end of this PR as I will need it

Reply:
![Screenshot 2024-01-24 at 15 15 35](https://github.com/leather-wallet/extension/assets/2938440/2429ab7d-ebf1-4fb7-9deb-67cf6ef4d3a5)

I got this working myself uglily and buggily but relatively easily. I'll iterate + improve

Reply:
Nice! 🚀